### PR TITLE
Created new SD theme called Midnight Qt

### PR DESCRIPTION
**Pull request by csharpee**
:ice_cream: :cake: :balloon:

### CODE REVIEW

**Comment by leto** [comment]
@csharpee this is really cool! Can you attach a screenshot of what it looks like?

**Comment by csharpee** [comment]
Screenshot_Midnight_A

Screenshot_Midnight_B

**Comment by csharpee** [comment]
The core background color has changed to create less stress on the eyes. The background is #111, rather than #333. For desktop dark themes, the color creates more harmony between other applications, including increase of creativity & less distraction of items.

The tabs for selecting screens are larger for ease of use and has less contrast between other items; the buttons 'feel' and logically are a part of the whole structure.

Slightly more distinguishing character and contrast between important items; ie. input box backgrounds.

Overall, less strain & distractions on the eyes; more direct feedback with buttons & highlighting of menu items; more harmony with other dark theme applications; other fine details.

> "A painter should begin every canvas with a wash of black, because all things in nature are dark except where exposed by the light." --Leonardo da Vinci


@everyone If y'all have any feedback or ideas, I am open to receiving them.

@Dukeleto,  if you don't want extra themes, we could put this in place of the dark theme.

**Square Snippets:**
+Larger Tabs
+Comfy Spacing
+Highlighting of Menu Items
+Highlighting of Push Buttons
+Border Colors of Transactions
+Bold of checkbox titles
+Checkbox/Radio buttons background change

Hush_Dark_Midnight_Comparison_AA

Hush_Dark_Midnight_Comparison_BB

Hush_Dark_Midnight_Comparison_CC

Hush_Dark_Midnight_Comparison_DD

File: src/settings.ui [hunk 1]
```diff
@@ -26,7 +26,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>3</number>
+      <number>1</number>
```
Comment (leto) [inline src/settings.ui:29]: @csharpee why is this change needed?

File: src/connection.cpp [hunk 2]
```diff
@@ -23,7 +23,7 @@ ConnectionLoader::ConnectionLoader(MainWindow* main, RPC* rpc) {
     QMovie *movie2 = new QMovie(":/img/res/silentdragon-animated-startup-dark.gif");;
     auto theme = Settings::getInstance()->get_theme_name();
     auto size  = QSize(512,512);
-    if (theme == "dark") {
+    if (theme == "dark" || "midnight") {
```
Comment (leto) [inline src/connection.cpp:26]: if you search src/*.cpp you will see there are 2 other places that need this same change

File: src/settings.ui (hunk 1 above)
Comment (csharpee) [reply src/settings.ui:29]: I didn't change it, for some reason clicking tabs changes the number in Qt. I can manually edit this with Geany :genie: ..Qt doesn't let us manually edit the code

File: src/connection.cpp (hunk 2 above)
Comment (csharpee) [reply src/connection.cpp:26]: Okay, dokey; working on this now. Thanks for the review @leto

**Comment by leto** [comment]
@csharpee thanks for making those changes! Merged. Into the Midnight Future we go...
